libs/community/langchain_community/vectorstores/vectara.py

```diff
@@ -317,69 +363,101 @@ def add_texts(
             )
         return [doc_id]
 
-    def vectara_query(
```

Comment:
any reason someone would have used this before?

Reply:
Not at all. I think this one is safe to change.